corehq/apps/userreports/models.py

```diff
@@ -66,6 +65,10 @@ class ElasticSearchIndexSettings(DocumentSchema):
     number_of_shards = IntegerProperty(default=2)
 
 
+class SQLColumnIndexes(DocumentSchema):
+    column_ids = StringListProperty()
```

Comment:
I made this a separate schema so it could later be modified to do partial queries or maybe a GIN index if we would like